README.md

```
@@ -30,26 +30,39 @@ the client, signaling that it should use SPNEGO protocol for authentication.
 
 > [!NOTE]
 > A lot of the behavior of this authentication backend is currently hardcoded
-> to only work with Red Hat systems.
-
-> [!NOTE]
-> Anonymous user access to the LDAP server is required for querying user
-> information.
+> to only work with Red Hat IPA LDAP systems (`dc=ipa,dc=redhat,dc=com`).
 
```

markdownlint-cli2 (0.23.2)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)